consent-protocol/tests/test_kai_voice_rollout_guardrails.py

```
@@ -0,0 +1,299 @@
+        return None
+
+    db_conn_stub.get_pool = _noop_get_pool
+    sys.modules["db.connection"] = db_conn_stub
+
+if "google" not in sys.modules:
+    sys.modules["google"] = types.ModuleType("google")
+if "google.genai" not in sys.modules:
+    sys.modules["google.genai"] = types.ModuleType("google.genai")
+if "google.genai.types" not in sys.modules:
+    sys.modules["google.genai.types"] = types.ModuleType("google.genai.types")
+setattr(sys.modules["google"], "genai", sys.modules["google.genai"])
```

Ruff (0.15.5)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)